Assets/_Functionalities/OGC3DTiles/Scripts/Tile3DLayerGameObject.cs

```diff
@@ -13,32 +14,29 @@
 
 namespace Netherlands3D.Functionalities.OGC3DTiles
 {
+    [RequireComponent(typeof(ReadSubtree))]
     [RequireComponent(typeof(Read3DTileset))]
-    public class Tile3DLayerGameObject : LayerGameObject, ILayerWithPropertyData, ILayerWithPropertyPanels, ILayerWithCredentials
+    public class Tile3DLayerGameObject : LayerGameObject, ILayerWithPropertyData, ILayerWithPropertyPanels// ILayerWithCredentials
     {
         public override BoundingBox Bounds => tileSet.root != null ? new BoundingBox(tileSet.root.BottomLeft, tileSet.root.TopRight) : null;
-        
+        public Tile3DLayerPropertyData PropertyData => urlPropertyData;
+
         private Read3DTileset tileSet;
         [SerializeField] private bool usePropertySections = true;
         private List<IPropertySectionInstantiator> propertySections = new();
 
         private Tile3DLayerPropertyData urlPropertyData;
         LayerPropertyData ILayerWithPropertyData.PropertyData => urlPropertyData;
-        public UnityEvent<string> OnURLChanged => urlPropertyData.OnUrlChanged;
+        
+        [Obsolete("this is a temporary fix to apply credentials to the 3d Tiles package. this should go through the ICredentialHandler instead")]
```

Comment:
Is dit nog steeds waar? En moet dit niet in deze story opgelost zijn, of moet er een aparte story voor komen om dit te addresseren?